### PR TITLE
Clean up HTML for `donate` app #2691

### PR DESCRIPTION
**Pull request by krsion**
Select of amount is now defined in Django, not in template.

### CODE REVIEW

File: readthedocs/donate/forms.py [hunk 1]
```diff
@@ -61,6 +61,27 @@ class Meta:
...
+        (800, "2 Days ($800)"),
+        (1200, "3 Days ($1200)"),
+        (1600, "4 Days ($1600)"),
+        (2000, "5 Days ($2000)"),
+        (4000, "2 Weeks ($4000)"),
+        (6000, "3 Weeks ($6000)"),
+        (8000, "4 Weeks ($8000)"),
+    ]
+    dollars_select = forms.ChoiceField(choices=amounts, initial=50,
+                                       widget=select_widget, label="Amount")
```
Comment (agjohnson) [inline readthedocs/donate/forms.py]: A couple style notes here:

* Stylistically, we support either `amounts` as a class-level constant (`SupporterForm.DOLLAR_AMOUNTS`), or in a `constants` module (`readthedocs.donate.constants.DOLLAR_AMOUNTS`).
* `select_databind` and `select_widget` shouldn't be defined as variables, and should be defined in the same spot as the other bindings, in `Meta`
* Translation was lost on these strings

File: readthedocs/donate/templates/donate/create.html [hunk 2]
```diff
@@ -71,27 +71,12 @@
...
-            <option value="4000">{% trans "2 Weeks" %} ($4000)</option>
-            <option value="6000">{% trans "3 Weeks" %} ($6000)</option>
-            <option value="8000">{% trans "4 Weeks" %} ($8000)</option>
-          </select>
+          {{ form.dollars_select }}
           {% if field.help_text %}
             <span class="helptext">{{ field.help_text }}</span>
           {% endif %}
         </p>
+      {% elif field.name == 'dollars_select' %}
```
Comment (agjohnson) [inline readthedocs/donate/templates/donate/create.html]: What purpose does this conditional serve?

Comment (krsion) [inline readthedocs/donate/templates/donate/create.html]: Select would be drawn twice if it wasn't there.

**Comment by ericholscher** [comment]
This code has been removed from the codebase, since it isn't really useful for OSS users -- thanks for the contributions!

**Pull request closed without merging**